nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/test/resources/templates/dummyNotificationToSendMail.ftl

```diff
@@ -0,0 +1,110 @@
+<html>
+<head>
+    <style>
+        div {
+            margin: 0;
+            padding: 0;
+            background-color: #e9ecef;
+            font-family: Arial, sans-serif;
+            marginheight: 0;
+            marginwidth: 0;
+        }
+
+        table {
+            margin: 0;
+            padding: 0;
+            border: none;
+        }
+
+        td {
+            border-collapse: collapse;
+            margin: 0;
+            padding: 20px;
+            border-top: 0;
+            align-content: center;
+            vertical-align: top;
+            border: 1px solid #eee;
+            color: #000;
+            font-size: 13px;
+        }
+
+        p {
+            font-size: 13px;
+        }
+
+        .propertyName {
+            color: #888;
+        }
+
+        .a {
+            color: #22aee8;
+            text-decoration: underline;
+            word-wrap: break-word !important;
+        }
+    </style>
+</head>
+<body>
+<div>
+    <br/>
+    <p>The document ${htmlEscape(docTitle)} is now available.</p>
+    <table>
+        <tbody>
+        <tr>
```

Comment:
As it's a dummy you don't need complex formatting, the requested strings could be one after the other.

Reply:
I agree, no need to have real content of the template right?
This might simplify `getMailContent()`, not needing a special rendering.

Reply:
ok for making the HTML very simple it's what i will do but if you agree i want to keep the dummy html file and i will have a separate expected mail content (only text of html tags) the same way i idid [here](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/test/java/org/nuxeo/ecm/platform/comment/CommentUtils.java#L136) my idea is the keep the things near the real onces